packages/compass-schema-validation/src/components/sample-documents/sample-documents.jsx

```diff
@@ -75,17 +106,20 @@ class SampleDocuments extends Component {
    * @returns {React.Component} The component.
    */
   renderMatchingDocuments() {
-    const title = 'Sample document that passed validation';
+    const {
+      sampleDocuments: { validDocument, validDocumentState },
+    } = this.props;
 
     return (
       <div className={sampleDocumentStyles} data-testid="matching-documents">
         <div className={cx(documentHeadingStyles, matchingStyles)}>
-          <Icon glyph="InfoWithCircle" size="small" />
-          <Body className={documentHeadingTextStyles}>{title}</Body>
+          <Icon glyph="CheckmarkWithCircle" size="small" />
+          <Body className={documentHeadingTextStyles}>Passed validation</Body>
         </div>
         <DocumentPreview
-          document={this.props.sampleDocuments.matching}
-          isLoading={this.props.sampleDocuments.isLoading}
+          document={validDocument}
+          loadingState={validDocumentState}
+          onLoadSampleClick={this.props.fetchValidDocument}
```

Comment:
This, and other document meta doesn't seem to be used outside of the `DocumentPreview`, might make sense to just have two separately connected components? This aligns with [redux styleguide suggestion](https://redux.js.org/style-guide/#connect-more-components-to-read-data-from-the-store) too a bit more

Reply:
SampleDocuments now render separately connected components